app/lib/configuration.js

```diff
@@ -20,6 +20,7 @@ class Configuration {
     this.theme = ''
     this.hotkey = ''
     this.debug = false
+    this.height = 700
   }
 
   load () {
```

Comment:
I presume you meant by this that I should be assigning `this.height = 700` in the load function, correct? I made a commit of exactly that now.